azurerm/helpers/validate/url_test.go

```diff
@@ -0,0 +1,45 @@
+package validate
+
+import (
+	"testing"
+)
+
+func TestHelper_Validate_UrlIsHttpOrHttps(t *testing.T) {
```

Comment:
I would just call this `Test<function name>` and if its one of many methods `Test<function name>_<description>`